libs/market-list/src/lib/components/select-market-columns.tsx

```diff
@@ -22,6 +22,7 @@ import { totalFees } from '../utils';
 import type { CandleClose } from '@vegaprotocol/types';
 import type { MarketList_markets_fees_factors } from '../__generated__/MarketList';
 import classNames from 'classnames';
+import { string } from 'zod';
```

Comment:
```suggestion
```

Reply:
I don't think you'd be using this anywhere, perhaps added by mistake?

Reply:
Yep, removed